src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter((person) => person.sex === 'm');
+
+  // eslint-disable-next-line max-len
+  const sorted = men.filter((person) => century === undefined ? true : century === Math.ceil(person.died / 100));
```

Comment:
Use a single filter statement instead of chaining multiple filter statements. You can use the && operator in the callback function to simplify the logic.